server/src/main/java/io/druid/client/cache/CacheMonitor.java

```diff
@@ -38,20 +39,30 @@ public CacheMonitor(
     this.cache = cache;
   }
```

Comment:
not within Druid at least. Fine to remove, unless we want to preserve binary compatibility for extensions that might bind it